Source/AlchemyLanguageV1/AlchemyLanguageV1.swift

```diff
@@ -0,0 +1,1613 @@
+            let failureReason = "Profile could not be encoded."
+            let userInfo = [NSLocalizedFailureReasonErrorKey: failureReason]
+            let error = NSError(domain: errorDomain, code: 0, userInfo: userInfo)
+            throw error
+        }
+        return body
+    }
+    
+    public func getAuthorsURL(
+        url: String,
+        failure: (NSError -> Void)? = nil,
+        success: DocumentAuthors -> Void)
+    {
+        
+        // construct request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceUrl + "/url/URLGetAuthors",
+            acceptType: "application/json",
+            contentType: "application/x-www-form-urlencoded",
+            queryParameters: [
+                NSURLQueryItem(name: "url", value: url),
+                NSURLQueryItem(name: "apikey", value: apiKey),
+                NSURLQueryItem(name: "outputMode", value: "json")
+            ]
+        )
+        
+        // execute request
+        Alamofire.request(request)
+            .responseObject(dataToError: dataToError) {
+                (response: Response<DocumentAuthors, NSError>) in
+                switch response.result {
+                case .Success(let authors): success(authors)
+                case .Failure(let error): failure?(error)
+                }
+        }
+        
+    }
+    
+    public func getAuthorsHtml(
+        html: NSURL,
+        url: String? = nil,
+        failure: (NSError -> Void)? = nil,
+        success: DocumentAuthors -> Void)
+    {
+        // construct body
+        let body = try? buildBody(html, html: true)
+        
+        // construct query paramerters
+        var queryParams = [NSURLQueryItem]()
+        
+        queryParams.append(NSURLQueryItem(name: "apikey", value: apiKey))
+        queryParams.append(NSURLQueryItem(name: "outputMode", value: "json"))
+        if let myUrl = url {
+            queryParams.append(NSURLQueryItem(name: "url", value: myUrl))
+        }
+        
+        // construct request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceUrl + "/html/HTMLGetAuthors",
+            contentType: "application/x-www-form-urlencoded",
+            messageBody: body,
+            queryParameters: queryParams
+        )
+        
+        // execute request
+        Alamofire.request(request)
+            .responseObject(dataToError: dataToError) {
+                (response: Response<DocumentAuthors, NSError>) in
+                switch response.result {
+                case .Success(let authors): success(authors)
+                case .Failure(let error): failure?(error)
+                }
+        }
+    }
+    
+    public func getRankedConceptsURL(
+        url: String,
+        knowledgeGraph: Bool? = false,
+        failure: (NSError -> Void)? = nil,
+        success: ConceptResponse -> Void)
+    {
+        let graph = shouldHaveField(knowledgeGraph!)
```

Comment:
Not necessarily, however I thought it made more sense from a usability standpoint. For the optional values and I felt it made more sense to ask for a boolean instead of an int (`1`) the service actually requires.